### PR TITLE
Add HTML <title> tags, and some fine-tuning

### PR DESCRIPTION
**Pull request by bonzini**
Fixes issue #43. After this one, there's only one series to go before I have nothing else on my plate!

(Take your time after new year is over of course!)

### CODE REVIEW

File: www/templates/project-list.html [hunk 1]
```diff
@@ -6,6 +6,11 @@
 
 {% block content %}
 
+<div class="row">
+ <div class="col-lg-12">
+  <h2>{% block title %}Project list{% endblock %}</h2>
+ </div>
+</div>
```
Comment (famz) [inline www/templates/project-list.html]: I prefer we don't add this block. The spacing, alignment, and sizing don't feel good in the surroundings.

Comment (famz) [reply www/templates/project-list.html]: I mean no `<h2>` in `<body>`. It's fine in `<head><title>...`.

**Comment by famz** [comment]
The rendering isn't quite right, I'm seeing white borders:

a

File: www/templates/series-detail.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
 <script src="/static/js/jquery.colorbox-min.js"></script>
 {% endblock %}
 
-{% block title %}{{ stripped_subject }}{% endblock %}
+{% block title %}{{ stripped_subject }}{% if has_other_revisions %} (v{{ version }}){% endif %}{% endblock %}
```
Comment (famz) [inline www/templates/series-detail.html]: Can we put this before the stripped_subject in brackets so it better matches the order and format in the original subject?

```
{% block title %}{% if has_other_revisions %}[v{{ version }}] {% endif %}{{ stripped_subject }}{% endblock %}
```

**Comment by bonzini** [comment]
I will apply the changes you suggest and check the borders on Chrome. The last patch can use a separate pull request. (Did you remember to run ./manage.py collectstatic?)

**Comment by famz** [comment]
> (Did you remember to run ./manage.py collectstatic?)

Now I remember but it doesn't help.

**Comment by famz** [comment]
> (Did you remember to run ./manage.py collectstatic?)

Good point. Now I do it and a full refresh, the view is correct.

**Pull request merged**